dev_env.tf

```diff
@@ -3,5 +3,22 @@ module "dev1_network" {
   source       = "./modules/network"
   network_name = "dev1"
   vpc_cidr     = "10.1.0.0/16"
+  # Save money: disable while no EC2s are deployed
+  internet_gateway = 0
+  nat_gateway = 0
 }
 
+module "dev2_network" {
+  source       = "./modules/network"
+  network_name = "dev2"
+  vpc_cidr     = "10.2.0.0/16"
+  # Save money: disable while no EC2s are deployed
+  internet_gateway = 0
+  nat_gateway = 0
+}
+
+module "infra_network" {
+  source       = "./modules/network"
+  network_name = "dev2"
```

Comment:
should probably call this `infrastructure` instead of `dev2`

Reply:
ah good catch

Reply:
opted for the shorter "infra" name. It will flow into other tags and it's nice to keep those short